src/main/resources/xslt-files/EMD_doi_datacite_v4.1.xsl

```diff
@@ -828,18 +828,25 @@
                 </xsl:choose>
             </xsl:element>
         </xsl:if>
-
-        <xsl:if test="$licenses">
+        
+        <xsl:variable name="cc0" select="'http://creativecommons.org/publicdomain/zero/1.0'"/>
+        <xsl:variable name="dansLicense" select="'https://dans.knaw.nl/en/about/organisation-and-policy/legal-information/DANSLicence.pdf'"/>
+        <xsl:if test="$licenses">            
             <xsl:for-each select="$licenses">
                 <xsl:choose>
-                    <xsl:when test=". = 'accept' and ../dcterms:accessRights = 'OPEN_ACCESS'">
-                        <xsl:variable name="cc0" select="'http://creativecommons.org/publicdomain/zero/1.0'"/>
+                    <xsl:when test=". = 'accept' and count($licenses) = 1">  
                         <xsl:element name="rights">
-                            <xsl:attribute name="rightsURI" select="$cc0"/>
-                            <xsl:value-of select="concat('License: ', $cc0)"/>
+                            <xsl:if test="$access-rights-datacite = 'OPEN_ACCESS'">
+                                <xsl:attribute name="rightsURI" select="$cc0"/>
+                                <xsl:value-of select="concat('License: ', $cc0)"/>
+                            </xsl:if>
+                            <xsl:if test="not($access-rights-datacite = 'OPEN_ACCESS')">
```

Comment:
Looks like this `test` is the negation of the previous test. Sounds like you want if-else? Not sure if that exists in XSLT?